test/views/ChoiceViewSpec.scala

```diff
@@ -67,6 +67,8 @@ class ChoiceViewSpec extends UnitViewSpec with CommonMessages with Stubs with In
                                     |list-of-available-journeys="CRT"
                                     |google-analytics.token=N/A
                                     |google-analytics.host=localhostGoogle
+                                    |gtm.container=a
+                                    |tracking-consent-frontend.url="http://localhost:6796/tracking-consent/tracking.js"
```

Comment:
For Movements you've used port `12345`. Which one is correct?

Reply:
yes you are correct

Reply:
fixed